README.md

```diff
@@ -184,6 +184,12 @@ block associated with the signature.
 Callback is called with `(err, success)` where success is true only if the signature is
 correct.
 
+#### `feed.commend(index, callback)`
```

Comment:
I'm not sure what 'commend' means personally =S  I might call the function 'rootHashes' if nobody else cares !

Reply:
Yes I was just about to comment that, rootHashes is better. I had to lookup commend in the dictionary.